### ISSUE TITLE
Search bar on package page should have a more consistent look with the search bar on the front page

### ISSUE THREAD

**Issue by szakarias** [dart-lang/pub-dev#7250]
e.g.
 - more oval search bar
 - placeholder text could be updated to be the same : "Search packages"
 - consider moving icon to the left
